jujugui/static/gui/src/test/test_environment_view.js

```diff
@@ -1003,15 +1003,6 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
                    'service does not have the ' + cssClass + ' class');
     }
 
-    it('should fade services and relations', function(done) {
```

Comment:
We do fade but this check is to test the relation line and the service fades in a linked way but that is no longer that case.